_shared/project/tox.ini

```diff
@@ -25,13 +25,22 @@ skip_install =
 setenv =
     PYTHONUNBUFFERED = 1
     OBJC_DISABLE_INITIALIZE_FORK_SAFETY = YES
+    dev: DEV = {env:DEV:true}
+    dev: SENTRY_ENVIRONMENT = {env:SENTRY_ENVIRONMENT:dev}
+    dev: NEW_RELIC_APP_NAME = {env:NEW_RELIC_APP_NAME:{{ cookiecutter.slug }}}
+    dev: NEW_RELIC_ENVIRONMENT = {env:NEW_RELIC_ENVIRONMENT:dev}
```

Comment:
Some common envvars that seem worth moving into the cookiecutter. I'm not sure that the Sentry and New Relic ones are actually needed in all projects (non-Pyramid apps, packages) but it won't do any harm to have them here.